pkg/apis/v1alpha5/suite_test.go

```diff
@@ -567,6 +567,40 @@ var _ = Describe("Provisioner Annotation", func() {
 		}
 		provisioner = test.Provisioner(testProvisionerOptions)
 	})
+	DescribeTable(
+		"Static Hash Values",
```

Comment:
Just for sanity checking and code coverage, can we ensure that changing the rest of the fields in the spec get a specific hash value as well?